src/service/mercury/queries.ts

```diff
@@ -0,0 +1,104 @@
+export const mutation = {
+  authenticate: `
+    mutation Auth {
+      authenticate(input: {email: $email, password: $password}) {
+        jwtToken
+      }
+    }
+  `,
+  newAccountSubscription: `
+    mutation MyMutation {
+      createFullAccountSubscription(
+        input: {fullAccountSubscription: {$pubKey: PUBKEY!, userId: 1}}
+      ) {
+        clientMutationId
+      }
+    }
+  `,
+};
+export const query = {
+  subscriptionById: `
+    query GetSubById {
+      contractEventById($id: ID!) {
+        id
+        data
+        contractId
+        ledgerTimestamp
+        nodeId
```

Comment:
It's unlikely you'll need the `nodeId`

Reply:
Also, it seems that this wants to get the subscription by the id by it's getting a contract event by the id, not the subscription.

Reply:
I actually won't need this one since as you mentioned in the other comment, it's easier to set up the queries by topic/publicKey. I removed the query in 45acdd6c7a529bc72912d87b02fc1873dcd37c80